zh-cn/index.html

```diff
@@ -0,0 +1,154 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>不要为了提问而提问</title>
+  <meta property="og:title" content="Don't ask to ask, just ask" />
+  <meta property="og:type" content="website" />
+  <meta property="og:url" content="https://dontasktoask.com" />
+  <meta property="og:locale" content="zh_CN" />
+  <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Open+Sans">
```

Comment:
I think Open Sans doesn't have non-latin characters, so this should probably be Noto Sans SC. The Korean translation also uses Noto Sans KR.

```suggestion
  <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Noto+Sans+SC">
```